pkg/scaling/scalers_builder.go

```diff
@@ -175,6 +175,8 @@ func buildScaler(ctx context.Context, client client.Client, triggerType string,
 		return scalers.NewInfluxDBScaler(config)
 	case "kafka":
 		return scalers.NewKafkaScaler(config)
+	case "kafka-x":
```

Comment:
I find this one a bit odd, maybe we should change it to the suggestions we had on the issue.

Thoughts @kedacore/keda-maintainers ?

Reply:
This I was not sure what to exactly put, `kafka-experimental`? Or is that too verbose.

`kafka-go` may get some people confused about whether referring to the scaler or the library.

Reply:
Someone raised `apache-kafka` as an option in today's meeting.

Reply:
OK, as there are no other suggestions going with `apache-kafka`

Reply:
remember to update it 😄